open-api/rest-catalog-open-api.yaml

```diff
@@ -2106,6 +2213,32 @@ components:
           items:
             $ref: '#/components/schemas/PartitionStatisticsFile'
 
+    PlanTask:
+      description:
+        An opaque JSON object that contains information provided by the REST server,
+        to be utilized by clients for distributed table scan planning, should be supplied
+        as is for input in `PlanTable` operation.
```

Comment:
when wouldn't this be supplied as-is?

Reply:
My understanding is that the `plan-task` should always be supplied "as is" for plan table.
cc @rdblue @danielcweeks @jackye1995

Reply:
I think Eduards point is we don't need to qualify by saying `as is`.  Just say supplied as input.

Reply:
will fix